### PR TITLE
15310 - text update for continuation in application

### PR DESCRIPTION
**Pull request by ketaki-deodhar**
*Issue #:*
https://github.com/bcgov/entity/issues/15310

*Description of changes:*

- text update for continuation in application. Addressing comments in PR#[2774](https://github.com/bcgov/sbc-auth/pull/2774)
- update unit tests
- verify `supported-continuation-in-entities` FF works as expected

Temporary Url for review: https://bcregistry-account-dev--pr-2775-ybj8049b.web.app/

Tested this just by flipping the FF value. 

image

image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2775-ybj8049b.web.app

**Review by vysakh-menon-aot — APPROVED** [review]

File: auth-web/src/locales/en.json [hunk 1]
```diff
@@ -206,10 +206,10 @@
...
-  "removeContinuationInSuccessTitle": "Continuation In Deleted",
-  "removeContinuationInSuccessText": "You have successfully deleted a continuation in",
-  "removeContinuationInConfirmTitle": "Delete Continuation In?",
-  "removeContinuationInConfirmText": "Deleting this continuation in will remove the application from your Business Registry list. If this continuation in was associated with a Name Request, the Name Request can still be used to start a continuation in.",
+  "removeContinuationInSuccessTitle": "Continuation In Application Deleted",
+  "removeContinuationInSuccessText": "You have successfully deleted a continuation in application",
+  "removeContinuationInConfirmTitle": "Delete Continuation In Application?",
+  "removeContinuationInConfirmText": "Deleting this continuation in application will remove the application from your Business Registry list. If this continuation in was associated with a Name Request, the Name Request can still be used to start a continuation in.",
```
Comment (severinbeauvais) [inline auth-web/src/locales/en.json:212]: "continuation in" is used twice more in this paragraph. Do those need to be updated?

Generally, did you chat with Yui about all of these and which ones need to be updated?

**Review by severinbeauvais — APPROVED** [review]

File: auth-web/src/locales/en.json (hunk 1 above)
Comment (ketaki-deodhar) [reply auth-web/src/locales/en.json:212]: I will double check with him

**Comment by ketaki-deodhar** [comment]
Merging this now. I will create another PR if there are any more updates from Yui.

**Pull request merged**